scss/_git_repo.scss

```diff
@@ -1,26 +1,126 @@
 .git-repo-areas {
-  margin: auto;
+    margin: auto;
 }
 
 .git-repo {
-  .repo-name {
-    font-size: 1.2rem;
-  }
-  .link-out {
-    position: relative;
-    padding: 0.3rem 0.8rem;
-    display: inline-block;
-    vertical-align: top;
-    height: 1rem;
-    &:before {
-      position: absolute;
-      content: "";
-      border-right: 0.5rem solid $red;
-      border-bottom: 0.5rem solid transparent;
-      width: 0.5rem;
-      height: 0.5rem;
-      display: inline-block;
-      vertical-align: top;
+    border-bottom: 1px solid #dbdbdb;
+    margin-bottom: 2rem;
+    color: #565555;
+
+    .repo {
+        &__name {
+            font-size: 1.3rem;
+            font-weight: bold;
+            margin-bottom: 1rem;
+
+            a {
+                text-decoration: none;
+                color: #0980e8;
+            }
+
+            span {
+                font-size: 0.5rem;
+                cursor: pointer;
+            }
+
+        }
+
+        &__notifications {
+            margin: 0 0 1.325rem 0;
+            padding: 0 0 0 15px;
+
+            .close {
+                position: absolute;
+                margin: 3px 0 0 -15px;
+                color: #DB5A51;
+                font-size: 0.75rem;
+                cursor: pointer;
+            }
+
+            li {
+                padding: 0;
+                margin: 0;
+                list-style: none;
+
+                .type {
+                    width: 100%;
+                    display: inline-block;
+                    color: #DB5A51;
+                    font-size: 0.525rem;
+                }
+
+                .title {
+                    width: 100%;
+                    display: inline-block;
+                    font-size: 0.75rem;
+                }
+
+            }
+
+        }
+
+        &__description {
+            font-size: 1rem;
+            margin-bottom: 1rem;
+        }
+
+        &__info {
+            font-size: 0.8rem;
+            margin-bottom: 1rem;
+
+            &--spacer {
+                padding-left: 0.25rem;
+                padding-right: 1rem;
+            }
+
+            &--footer {
+                display: inline;
+
+                svg {
+                    position: relative;
+                    height: 16px;
+                    width: 10px;
+                    top: 3px;
+                }
+
+                .repo-icon {
+                    position: relative;
+                    top: 1px;
+                    display: inline-block;
+                    width: 12px;
+                    height: 12px;
+                    border-radius: 50%;
+                    margin-right: 0.3rem;
+                }
+
+                .repocolor {
```

Comment:
following the class names you have, i think this would be more suitably set as `.repo-color`